### ISSUE TITLE
[Feature] Option to toggle haptic vibrations in app

### ISSUE THREAD

**Issue by evancramb** [dangwu/Octal#217]
Big fan of the app so far, but I find the vibration that occurs when expanding or collapsing comments a bit annoying. Could you add a toggle switch to the settings tab to disable this? I am able to disable it for my entire phone, but would rather turn it off just in Octal. Thanks!

**Comment by dangwu** [comment]
Yeah I can disable it for comment expand/collapse You're the second person to mention it.
